tests/lib/tools/fs-state

```diff
@@ -1,38 +1,53 @@
 #!/bin/bash
 
+show_help() {
+	echo "usage: fs-state wait-for-file [filename] [iterations] [sleep-time]"
+	echo "usage: fs-state ensure-dir-exists [direname]"
+	echo "usage: fs-state ensure-dir-exists-backup-real [direname]"
+	echo "usage: fs-state clean-dir [direname]"
+	echo "usage: fs-state ensure-file-exists [filename]"
```

Comment:
It's unclear how this differs from `touch`.

Reply:
mock_file just creates a file with that name in case it does not exist, mock_file_backup_real backups the file in case it already exists. Then restore-file restores both commands to the initial situation.

I think is it possible to get rid of  mock_file_backup_real and make mock-file to auto backup is it is needed, @zyga does it make sense? The same will apply for directories

Reply:
Yeah, I think this is fine.